src/helm/benchmark/run_specs.py

```diff
@@ -2550,7 +2569,9 @@ def construct_run_specs(spec: ObjectSpec) -> List[RunSpec]:
 
     def alter_run_spec(run_spec: RunSpec) -> RunSpec:
         try:
-            model = get_model(run_spec.adapter_spec.model)
+            deployment: ModelDeployment = get_model_deployment(run_spec.adapter_spec.model_deployment)
```

Comment:
I think you should do the default `model_deployment` thing here - i.e. if `model_deployment` in `AdapterSpec` is empty, set it to the same value as `model`.

But let me know if you have a better place to add this default.

Reply:
See the example below, it can break backward compatibility. For example `lmsys/vicuna-7b-v1.3` became `together/vicuna-7b-v1.3`.